tasks/configure_namespace.yml

```diff
@@ -74,6 +74,21 @@
   - source-containers
 
 
+- name: Create configmaps
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'templates/reactor_config_map.yml') | from_yaml }}"
+    validate:
+      fail_on_error: no
+      strict: yes
+  with_items:
+    - { name: "reactor-config-map", allowed_domains: "{{ allowed_domains }}"}
+    - { name: "reactor-config-map-scratch", allowed_domains: "{{ allowed_domains_scratch }}"}
+  tags:
+  - configmap
+  - source-containers
```

Comment:
Generaly, the same role should be used also for binary containers, they should differ only in configmaps, secrets

Reply:
OK removed  tag